src/database/migrations/20240806044802_user.ts

```diff
@@ -0,0 +1,16 @@
+import type { Knex } from "knex";
+
+
+export async function up(knex: Knex): Promise<void> {
+    return knex.schema
+    .createTable('users', function (table) {
+      table.increments('id');
+      table.string('first_name', 255).notNullable();
+      table.string('last_name', 255).notNullable();
+    })
+}
+
+
+export async function down(knex: Knex): Promise<void> {
```

Comment:
You should put a down which would be the drop table command so when you rollback your migrations it removes the table

Reply:
command added